libs/fluid-elements/tab-group/src/lib/tab-events.ts

```diff
@@ -16,21 +16,27 @@
 
 /** Custom event implementation fires when the active tab has changes */
 export class FluidTabGroupActiveTabChanged extends CustomEvent<any> {
-  constructor(public activeTab: string) {
+  constructor(public activeTabId: string) {
     super('activeTabChanged', { bubbles: true, composed: true });
   }
 }
 
 /** Custom event implementation that fires when a tab is clicked providing the active tab id  */
 export class FluidTabActivatedEvent extends CustomEvent<any> {
-  constructor(public activeTab: string) {
+  constructor(public tabId: string) {
     super('tabActivated', { bubbles: true, composed: true });
   }
 }
 
 /** Custom event implementation that fires when a tab is disabled providing the disabled tab id  */
 export class FluidTabDisabledEvent extends CustomEvent<any> {
-  constructor(public disableTab: string) {
+  constructor(public tabId: string) {
     super('disabled', { bubbles: true, composed: true });
   }
 }
+
+export class FluidTabBlurredEvent extends CustomEvent<any> {
+  constructor(public tabId: string) {
+    super('blurred', { bubbles: true, composed: true });
```

Comment:
The native event would be called `blur`. 
As long as we can, we should stick to the native namings as close as possible. If it does not work when calling it blur, please use `tabBlurred` (as seen on `tabActivated`).